libcxx/test/std/utilities/expected/expected.expected/monadic/transform.pass.cpp

```diff
@@ -237,25 +237,25 @@ constexpr void test_sfinae() {
     return std::expected<int, int>();
   };
 
-  e1.transform(never_called);
-  std::move(e1).transform(never_called);
-  ce1.and_then(never_called);
-  std::move(ce1).transform(never_called);
+  (void)e1.transform(never_called);
+  (void)std::move(e1).transform(never_called);
+  (void)ce1.and_then(never_called);
+  (void)std::move(ce1).transform(never_called);
```

Comment:
The `and_then` here looks like a copy-paste mistake. I could change this to `transform` if anyone wants.

Reply:
Yes, please!